libs/ddhub-client-gateway-frontend/ui/gateway-settings/src/lib/components/SettingsItem/SettingsItem.tsx

```diff
@@ -6,16 +6,20 @@ interface SettingsItemProps {
   title: string;
   icon: string;
   buttonText: string;
+  buttonText2?: string;
   content: React.ReactElement;
   onClick: () => void;
+  onClick2?: () => void;
   footer?: React.ReactElement;
 }
 
 export const SettingsItem: FC<SettingsItemProps> = ({
   icon,
   title,
   buttonText,
+  buttonText2,
```

Comment:
```suggestion
  secondaryButton
```
and same with `secondaryOnClick` ? You can adjust as well buttonText to be `primaryButton`

Reply:
will make the changes in another PR (improvements)